index.html

```diff
@@ -184,5 +264,102 @@ <h4 class="top-banner__subtitle">
         </div>
       </div>
     </section>
+    <section class="why-us">
+      <div class="why-us__container">
+        <div class="why-us__main-content">
```

Comment:
I bet you can get rid of this wrapper without losing anything. Just a point for code optimization. Try to find a way to get the desired design without it.